packages/react-badge/src/components/CounterBadge/CounterBadge.types.ts

```diff
@@ -1,11 +1,17 @@
 import type { BadgeProps, BadgeState } from '../Badge/index';
 
-type CounterBadgeCommons = {
+export type CounterBadgeProps = Omit<BadgeProps, 'appearance' | 'shape' | 'color'> & {
```

Comment:
_nit:_ Since we're already ordering things alphabetically in this PR maybe we can do that here too

```suggestion
export type CounterBadgeProps = Omit<BadgeProps, 'appearance' | 'color' | 'shape'> & {
```